client/web-sveltekit/src/routes/search/SearchResults.svelte

```diff
@@ -72,6 +82,8 @@
     $: resultsToShow = results.slice(0, count)
     $: expandedSet = cacheEntry?.expanded || new Set<SearchMatch>()
 
+    $: previewResult = writable<ContentMatch | SymbolMatch | PathMatch | null>(cacheEntry?.preview ?? null)
```

Comment:
Do you need the type parameter? Shouldn't it infer it from `cacheEntry.preview`?
```suggestion
    $: previewResult = writable(cacheEntry?.preview ?? null)
```

Reply:
Sure enough. Fixed!